ExportData.cs

```diff
@@ -91,48 +79,24 @@ public static void Execute(IList<IList<object>> list)
             });
 
             // Define request parameters.
+            // @todo Fetch these from a config.json file or environment variables?
```

Comment:
create a class called `BinocularsConfig` that represents your settings (for example, it would have a field called `GoogleSpreadsheetId`). This can be stored in `json` files and easily read & `de/serialised` in the Extension constructor.

Reply:
Agreed, discussion continues as issue https://github.com/teamtreedyn/Binoculars/issues/12 therefore marking resolved.